pkg/utils/experimentHelpers.go

```diff
@@ -7,6 +7,34 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+func (expDetails *ExperimentDetails) SetValueFromChaosExperiment(clients ClientSets) {
+	expDetails.SetImage(clients)
+	expDetails.SetArgs(clients)
+	expDetails.SetLabels(clients)
+	// Generation of Random String for appending it into Job
+	randomString := RandomString()
+	// Setting the JobName in Experiment Realted struct
+	expDetails.JobName = expDetails.Name + "-" + randomString
+}
+func (expDetails *ExperimentDetails) SetENV(engineDetails EngineDetails, clients ClientSets) {
+	// Get the Default ENV's from ChaosExperiment
+	log.Infoln("Getting the Default ENV Variables")
+	expDetails.SetDefaultEnv(clients)
+	// OverWriting the Defaults Varibles from the ChaosEngine ENV
+	log.Infoln("Patching some required ENV's")
+	expDetails.SetEnvFromEngine(engineDetails.Name, clients)
+	// Adding some addition necessary ENV's
```

Comment:
Let the comment mention that these are obtained from the spec.appinfo of the engine.